daemon/device-plugin/deviceplugin.go

```diff
@@ -104,45 +115,40 @@ func (nf *nfResources) Allocate(ctx context.Context, rqt *pluginapi.AllocateRequ
 	return resp, nil
 }
 
-func (nf *nfResources) GetDeviceState(DeviceName string) string {
-	// TODO: Discover device health
-	return pluginapi.Healthy
-}
-
-func (nf *nfResources) Start() error {
-	nf.cleanup()
-	nf.ensureConnected()
-
-	ctx := context.Background()
+func (nf *nfResources) GetDevices() error {
+	err := nf.ensureConnected()
+	if err != nil {
+		return err
```

Comment:
can we wrap the return here? `fmt.Errorf("... : %v", err)`

Reply:
Done